### PR TITLE
Added some examples to the doc strings.

### PR DESCRIPTION
**Pull request by klangner**
This is related to the issue #618.

### CODE REVIEW

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -735,6 +747,16 @@ def reduce_min(input_tensor, reduction_indices=None, keep_dims=False,
...
   tensor with a single element is returned.
 
+  For example:
+
+  ```python
+  # 'x' is [[1, 2]
+  #         [3, 4]]
+  tf.reduce_min(x) ==> 1
+  tf.reduce_min(x, 0) ==> [1, 2]
+  tf.reduce_min(x, 1) ==> [1,  3]
```
Comment (martinwicke) [inline tensorflow/python/ops/math_ops.py]: Can you remove the second space before `3]`?

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -762,6 +784,16 @@ def reduce_max(input_tensor, reduction_indices=None, keep_dims=False,
...
   tensor with a single element is returned.
 
+  For example:
+
+  ```python
+  # 'x' is [[1, 2]
+  #         [3, 4]]
+  tf.reduce_max(x) ==> 4
+  tf.reduce_max(x, 0) ==> [3, 4]
+  tf.reduce_max(x, 1) ==> [2,  4]
```
Comment (martinwicke) [inline tensorflow/python/ops/math_ops.py]: Can you remove the second space before `4]`?

**Comment by klangner** [comment]
Done.

**Comment by martinwicke** [comment]
Can you squash the commits, then I'll merge.

**Comment by klangner** [comment]
It looks that I messed it up. I guess I better close this one :-(

**Comment by Mistobaan** [comment]
no need to close the pull request. Just fetch the code again and do something like 
```bash
git reset origin/master
git add  tensorflow/python/ops/math_ops.py
git commit -m "Adds some examples to the doc strings. See issue #618"
```

**Pull request closed without merging**